index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ну игры</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1> Топ игр жизни</h1>
+    </header>
+    <main>
+        <div class="center">
+            <section class="main-section">
+                <figure class="main-aba">
+                    <div class="main-abob">
+                        <img src="HK.jpg" alt="Hollow Knight" width="236" height="144">
```

Comment:
```suggestion
                        <img src="HK.jpg" alt="Hollow Knight" ">
```

`width="236" height="144` - не самый лучший вариант указания размеров картинок